### PR TITLE
9684 Message Display: Display Place and Date of Trial Session When Case Status = Calendared

### PR DESCRIPTION
**Pull request by mrinsin**
Reindex required! 
   - [x] staging
   - [x] irs
   - [x] dev
   - [x] prod
   - [x] test

image

### CODE REVIEW

File: shared/src/business/entities/Message.js [hunk 1]
```diff
@@ -17,37 +17,49 @@ function Message() {
 }
 
 Message.prototype.init = function init(rawMessage, { applicationContext }) {
+  messageDecorator(this, rawMessage, { applicationContext });
+};
+
+const messageDecorator = (
```
Comment (TomElliottFlexion) [inline shared/src/business/entities/Message.js:23]: This is nice, I like this

File: shared/src/business/utilities/abbreviateState.test.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+const { abbreviateState } = require('./abbreviateState');
+
+describe('abbreviateState', () => {
```
Comment (TomElliottFlexion) [inline shared/src/business/utilities/abbreviateState.test.js:3]: Might it be worth adding a test around multi-word cities? e.g. 'Las Vegas, Nevada'

Comment (mrinsin) [reply shared/src/business/utilities/abbreviateState.test.js:3]: Ooh thats a great idea, on it!

File: web-client/src/views/Messages/MessagesIndividualInbox.jsx [hunk 3]
```diff
@@ -157,7 +157,15 @@ export const MessagesIndividualInbox = connect(
                   <td className="message-queue-row max-width-25">
                     {message.caseTitle}
                   </td>
-                  <td className="message-queue-row">{message.caseStatus}</td>
+                  {!message.showTrialInformation && (
+                    <td className="message-queue-row">{message.caseStatus}</td>
```
Comment (TomElliottFlexion) [inline web-client/src/views/Messages/MessagesIndividualInbox.jsx:161]: These could be if...elses, but that's just a matter of preference

**Review by TomElliottFlexion — APPROVED** [review]

File: web-client/src/views/Messages/MessagesIndividualInbox.jsx (hunk 3 above)
Comment (mrinsin) [reply web-client/src/views/Messages/MessagesIndividualInbox.jsx:161]: We tried it as an if/else but didn't find it readable (to our eyes at least) so we decided to go down this route.. good suggestion tho!

**Review by mmarcotte — APPROVED** [review]

**Pull request merged**